fms2_io/netcdf_io.F90

```diff
@@ -61,9 +81,10 @@ module netcdf_io_mod
   class(*), dimension(:,:,:,:,:), pointer :: data5d => null() !< 5d data pointer.
   logical :: was_read !< Flag to support legacy "query_initialized" feature, which
                       !! keeps track if a file was read.
+  logical :: is_bc_variable
+  type(bc_information) :: bc_info
```

Comment:
These variables should have descriptions

Reply:
Fixed in f0d7161f31478f7ef7f92b4eee505d409ef3b0eb